commit 648a79b994b41a1056886976735ff4b9bbb6ea2c
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 11:20:34 2010 -0700

    Staging: brcm80211: remove STATIC definition
    
    Use 'static', it's the correct thing to do.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/bcmutils.c
+++ b/drivers/staging/brcm80211/brcmfmac/bcmutils.c
@@ -1049,7 +1049,7 @@ int bcm_iovar_lencheck(const bcm_iovar_t
  * ****************************************************************************
  */
 
-STATIC const u8 crc8_table[256] = {
+static const u8 crc8_table[256] = {
 	0x00, 0xF7, 0xB9, 0x4E, 0x25, 0xD2, 0x9C, 0x6B,
 	0x4A, 0xBD, 0xF3, 0x04, 0x6F, 0x98, 0xD6, 0x21,
 	0x94, 0x63, 0x2D, 0xDA, 0xB1, 0x46, 0x08, 0xFF,
@@ -1170,7 +1170,7 @@ uint16 hndcrc16(u8 *pdata,	/* pointer to
 	return crc;
 }
 
-STATIC const uint32 crc32_table[256] = {
+static const uint32 crc32_table[256] = {
 	0x00000000, 0x77073096, 0xEE0E612C, 0x990951BA,
 	0x076DC419, 0x706AF48F, 0xE963A535, 0x9E6495A3,
 	0x0EDB8832, 0x79DCB8A4, 0xE0D5E91E, 0x97D2D988,
--- a/drivers/staging/brcm80211/include/bcmdefs.h
+++ b/drivers/staging/brcm80211/include/bcmdefs.h
@@ -67,7 +67,6 @@
 #define BCMROMDAT_NAME(_data)	_data
 #define BCMROMFN(_fn)		_fn
 #define BCMROMFN_NAME(_fn)	_fn
-#define STATIC	static
 #define BCMROMDAT_ARYSIZ(data)	ARRAYSIZE(data)
 #define BCMROMDAT_SIZEOF(data)	sizeof(data)
 #define BCMROMDAT_APATCH(data)
--- a/drivers/staging/brcm80211/util/hnddma.c
+++ b/drivers/staging/brcm80211/util/hnddma.c
@@ -273,7 +273,7 @@ static bool dma64_rxstopped(dma_info_t *
 static bool dma64_rxenabled(dma_info_t *di);
 static bool _dma64_addrext(osl_t *osh, dma64regs_t *dma64regs);
 
-STATIC inline uint32 parity32(uint32 data);
+static inline uint32 parity32(uint32 data);
 
 const di_fcn_t dma64proc = {
 	(di_detach_t) _dma_detach,
@@ -612,7 +612,7 @@ dma32_dd_upd(dma_info_t *di, dma32dd_t *
 }
 
 /* Check for odd number of 1's */
-STATIC inline uint32 parity32(uint32 data)
+static inline uint32 parity32(uint32 data)
 {
 	data ^= data >> 16;
 	data ^= data >> 8;
